docs/current/742989-cookbook.md

```diff
@@ -625,6 +625,44 @@ Set the Hashicorp Vault URI, namespace, role and secret identifiers as host envi
 
 [Learn more](./guides/723462-use-secrets.md)
 
+
+## Error handling
+
+### Aborting with unhandled error
```

Comment:
```suggestion
### Abort pipeline with unhandled error
```

Reply:
Maybe just "Abort with unhandled error"?

Reply:
Modified

Reply:
is `Abort pipeline script with unhandled error` clearer?

Reply:
I was hoping abort was clear enough. Abort means quit with an error.

I'm ok with the change, but the word "pipeline" is loaded and we're still working out the terminology. While script is accurate in Python and Node.js, it's only that in Go when doing `go run`, but it's usually considered an app or program.That's why I just put "Abort", to avoid having to name things 😛

Reply:
FTR, in Go:

> Exit causes the current program to exit with the given status code. Conventionally, code zero indicates success, non-zero an error. The program terminates immediately; [...]

\- [os.Exit](https://pkg.go.dev/os#Exit)

Reply:
Going to remove.